src/client/index.ts

```diff
@@ -18,111 +79,268 @@ export interface NotificationOptions {
      * If not provided at time of creation, one will be generated for you and returned as part of the {@link create} method.
      */
     id?: string;
+
     /**
-     * Main Notification content.
+     * Title of the notification.
+     *
+     * Displayed as the first line of the notification, in a heading style.
      */
-    body: string;
+    title: string;
+
     /**
-     * Title of the Notification (e.g. sender name for email).
+     * Notification body text.
+     *
+     * This is the main notification content, displayed below the notification title. The notification will expand to fit the length of this text.
      */
-    title: string;
+    body: string;
+
     /**
-     * Subtitle of the Notification.
+     * Describes the context of this notification. Allows users to control different notification types that are raised
+     * by an application.
+     *
+     * This string is not displayed on the notification itself, but should be user-readable. It will be displayed in
+     * the notification center preferences section, and any string passed as a `category` should make sense when
+     * displayed in that context.
+     *
+     * In general, event categories should list all the different types of notifications raised by your app. As a
+     * general guide, if a user may want to have different preferences for some subset of notifications created by your
+     * app, then ensure those notifications have a distinct category.
+     *
+     * For example - given a calendar app, notification categories could be:
+     * - `"Upcoming Events"`: Notification that an event is about to start
+     * - `"Event Start"`: Notification raised when event starts, expiring at the event end time
+     * - `"Event Modified"`: When an event is modified
+     * - `"Event Cancelled"`: When an event is cancelled
+     * - `"Event Response"`: An attendee has responded to an event invitation that you created
+     * - `"Daily Agenda"`: A notification sent each morning with event reminders
+     *
+     * **NOTE:** The user-facing UI that a user would use to manage their preferences is still in progress. This
```

Comment:
They do need to worry about it though, as these strings will become user-visible when we put out a future release of the provider.

Want to clarify in the docs that this is something you should think about now and set properly, even though it won't actually do anything just yet (but it will soon, and notifications are persistent!).

Reply:
My point is that most users will likely not read that far down the comment and will find themselves wondering why this doesn't seem to do anything. Having it at the top works as "you should be aware of this, but it won't do anything right now", at least in my mind.